Makefile

```diff
@@ -0,0 +1,101 @@
+# RELOAD PROJECT MAKE UP
+# This file exemplifies the main commands that should be available in all
+# our project repos. The commands may function differently in each project,
+# but they should achieve the same goal.
+#
+# The commands are:
+#   make reset 	- stop all containers, update them and db dumps and start again
+#   make up 	- just bring up all containers
+#   make stop 	- stop all containers
+#   make test	- run any local tests
+#   make logs   - follow docker logs from the containers
+#   make status - show status of the environment
+
+# List of containers
+DOCKER_CONTAINERS = web fpm db mailhog
```

Comment:
Why do we need to keep a list of containers?

As I understand it running `docker-compose` commands without naming containers should apply the command to all containers.

Reply:
Yes, this comes from a use case where the Makefile targets are split into groups of containers that can be reset independently from each other (eg. a Drupal + React setup). But it's a bit specialized and out of scope for this example, so I removed it.